cli/slsa-verifier/main.go

```diff
@@ -38,7 +41,19 @@ func main() {
 		"print the verified provenance to std out")
 	flag.Parse()
 
-	if provenancePath == "" || artifactPath == "" || source == "" {
+	if (provenancePath == "" || artifactPath == "") && artifactImage == "" {
+		fmt.Fprintf(os.Stderr, "either 'provenance' and 'artifact-path' or 'artifact-image' must be specified\n")
+		flag.Usage()
+		os.Exit(1)
+	}
+
+	if artifactImage != "" && (provenancePath != "" || artifactPath != "") {
+		fmt.Fprintf(os.Stderr, "'provenance' and 'artifact-path' should not be specified when 'artifact-image' is provided\n")
```

Comment:
note: that won't be true for GCB-built containers. Not something for this PR, though.

Reply:
Interesting, OK. Maybe need to redefine the interface for `VerifyImage`